rskj-core/src/main/java/org/ethereum/net/rlpx/FrameCodecHandler.java

```diff
@@ -85,8 +85,7 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
             loggerNet.debug("FrameCodec failed: ", cause);
         } else {
             if (cause instanceof IOException) {
-                loggerNet.info("FrameCodec failed: " + ctx.channel().remoteAddress() + "(" + cause.getMessage() + ")");
-                loggerNet.debug("FrameCodec failed: " + ctx.channel().remoteAddress(), cause);
+                loggerNet.info("FrameCodec failed: {} ({})", ctx.channel().remoteAddress(), cause);
```

Comment:
I would recheck if passing a Throwable as last parameter for the logger methods doesn't have a different semantic in the terms of how this is resolved as part of the logging message

Reply:
Yes, I'll keep the trace with throwable as last parameter. It has another behavior :+1:

Reply:
I mentioned this b/c of the second `{}` in the message marker, I'm not sure if it'll use the _throwable_ as part of the message or will just make it available as `%ex`